packages/adblocker/test/parsing.test.ts

```diff
@@ -803,6 +803,25 @@ describe('Network filters', () => {
           denyallow: undefined,
         });
       });
+
+      context('to', () => {
+        it('fails when both denyallow and to used', () => {
+          network('||foo.com$denyallow=foo.com,to=bar.com', null);
+          network('||foo.com$to=foo.com,denyallow=bar.com', null);
```

Comment:
if both present, can we merge them instead of dropping?

Reply:
Yes, we can normalize them.

Reply:
lets try to do that please

Reply:
Updated in 35c36cc093fd3775d85d9b91cd44e0deebd8b5c1